src/lib/components/market/response_render_results_status.ts

```diff
@@ -0,0 +1,155 @@
+function overrideOnResponseRenderResults(proto: any) {
```

Comment:
Please define types for `g_oSearchResults` (see steam.d.ts) -- it doesn't have to be comprehensive but is certainly better than using any.